frontend/taipy-gui/src/components/Taipy/Chart.tsx

```diff
@@ -11,7 +11,7 @@
  * specific language governing permissions and limitations under the License.
  */
 
-import React, { CSSProperties, useCallback, useEffect, useMemo, useRef, useState, lazy, Suspense } from "react";
```

Comment:
It would be nice to have the same formatting rules ...
My VsCode prefers the space inside the curly brackets ;-)